### ISSUE TITLE
False positive on ldap.SCOPE_SUBTREE

### ISSUE THREAD

**Issue by perlun** [PyCQA/pylint#4414]
Hi,

I'm experiencing the following which _seems_ to be an issue with `pylint`. But feel free to point me another direction if I'm understanding this incorrectly. Thanks for a nice tool. :+1: :bow: 

### Steps to reproduce

Given a file `test.py`:
```python
# pylint: disable=missing-module-docstring

import ldap

print(ldap.SCOPE_SUBTREE)
```

### Current behavior

Result of `pylint test.py`:
```
************* Module test
test.py:5:6: E1101: Module 'ldap' has no 'SCOPE_SUBTREE' member (no-member)

----------------------------------------------------------------------
Your code has been rated at -15.00/10 (previous run: -20.00/10, +5.00)
```

### Expected behavior
`ldap.SCOPE_SUBTREE` definitely exists, but it's defined using some slightly magic (?) way that I don't fully understand: https://github.com/python-ldap/python-ldap/blob/2fb97af1e025a95d1ea86b9aeac6235358e52278/Lib/ldap/constants.py#L222

Here's the output from running the script in itself:

```shell
$ python3 test.py 
2
```

### pylint --version output

Result of `pylint --version` output:

```
pylint 3.0.0a3
astroid 2.5.6
Python 3.9.2 (default, Feb 28 2021, 17:03:44) 
[GCC 10.2.1 20210110]
```

Additional dependencies:
```
python-ldap==3.3.1
```

**Comment by hippo91** [comment]
@perlun thanks for your report. It seems to be a bug.

**Comment by mbyrnepr2** [comment]
python-ldap uses [C extensions](https://github.com/python-ldap/python-ldap/blob/master/setup.py#L104) & the false positive goes away with:
```python
pylint --extension-pkg-whitelist=_ldap example.py
```

@Pierre-Sassoulas what do you think we can do to make this clear to users - do we document commands such as this? 

Alternatively do you think there would still be value in creating a plugin for this?

**Comment by Pierre-Sassoulas** [comment]
There is a message that suggest to use ``extension-pkg-whitelist`` (``c-extension-no-member``) but it's never raised instead of no-member like it should it seems. We have a documentation
